test/integration/features/basic.feature

```diff
@@ -1,8 +1,31 @@
 @basic
 Feature: Basic
-  In order to use Minishift
-  As a user
-  I need to be able to bring up a test environment
+  As a user I can perform basic operations of Minishift and OpenShift
+
+  Scenario: User can install default add-ons
+    Given Minishift has state "Does Not Exist"
+    When executing "minishift addons install --defaults" succeeds
+    Then stdout should contain
+     """
+     Default add-ons anyuid, admin-user, xpaas installed
+     """
+
+  Scenario: User can enable the anyuid add-on
+    Given Minishift has state "Does Not Exist"
+    When executing "minishift addons enable anyuid" succeeds
+    Then stdout should contain
+     """
+     Addon 'anyuid' enabled
+     """
+
+  Scenario: User can list enabled plugins
+    Given Minishift has state "Does Not Exist"
+    When executing "minishift addons list" succeeds
+    Then stdout should contain
+     """
+     - anyuid         : enabled    P(0)
+     - admin-user     : disabled   P(0)
+     """
```

Comment:
@LalatenduMohanty only issue which I can see without addon is time taken to have pv created which is right now not predicted. ( @hferentschik tried with try and sleep time also). Another use oc command same way to wait till pv job is done `oc get job persistent-volume-setup -n default -o 'jsonpath={ .status.active }'`